.github/workflows/workflow.yaml

```diff
@@ -78,6 +78,14 @@ jobs:
       - name: Run script
         run: |
           make require-suites
+  require_sync:
+    runs-on: ubuntu-latest
+    name: "Require a sync.yaml file and metadata.gatekeeper.sh/requiresSyncData annotation for every template.yaml using data.inventory"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run script
+        run: |
+          make require-sync
```

Comment:
if this fails, can we output an actionable message to the user

Reply:
Yes, `require-sync` will return a message with the template and what is missing the user needs to add:

`template at path '%s' is missing annotation with key '%s'`
or
`sync.yaml not found in dir '%s'`